results_test.go

```diff
@@ -5,29 +5,29 @@
 package revel
 
 import (
+	"fmt"
 	"net/http/httptest"
 	"strings"
 	"testing"
-	"fmt"
 )
 
 // Added test case for redirection testing for strings
 func TestRedirect(t *testing.T) {
 	startFakeBookingApp()
-	redirect:= RedirectToURLResult{fmt.Sprintf("/hotels/index/foo")}
+	redirect := RedirectToURLResult{fmt.Sprintf("/hotels/index/foo")}
 	resp := httptest.NewRecorder()
-	c := NewTestController(resp,showRequest)
-	redirect.Apply(c.Request,c.Response)
+	c := NewTestController(resp, showRequest)
+	redirect.Apply(c.Request, c.Response)
 	if resp.Header().Get("Location") != "/hotels/index/foo" {
-		t.Error("Failed to set redirect header correctly. : %s", resp.Header().Get("Location"))
+		t.Errorf("Failed to set redirect header correctly. : %s", resp.Header().Get("Location"))
```

Comment:
NOTE: This is strictly unrelated to the current PR. It was broken, not sure why it has gone unnoticed.